### PR TITLE
actually trying to address the issue... by just retrying

### PR DESCRIPTION
**Pull request by cviecco**

### CODE REVIEW

File: lib/client/twofa/u2f/u2f.go [hunk 1]
```diff
@@ -154,13 +155,28 @@ func doU2FAuthenticate(
...
 	var rawBytes []byte
 	for {
 		res, err := t.Authenticate(req)
 		if err == u2ftoken.ErrPresenceRequired {
 			time.Sleep(200 * time.Millisecond)
 			continue
 		} else if err != nil {
+			if runtime.GOOS == "darwin" && retryCount < 3 {
+				retryCount += 1
+				if err.Error() == "hid: general error" {
```
Comment (erikespinoza) [inline lib/client/twofa/u2f/u2f.go:168]: might it make sense to add something like `|| err.Error() == "hid: privilege violation"`" to retry on the issue that is reported in https://github.com/Cloud-Foundations/keymaster/issues/2

**Review by eespinozans1 — APPROVED** [review]
Tested on YubiKey 5C NFC. Haven't had an issue yet.

**Comment by jrusso1020** [comment]
Is there any update on this PR? We have been encountering these errors ourselves and would look forward to using this update

**Comment by rgooch** [comment]
@jrusso1020: Merged!

**Pull request merged**